openapi/src/parts/folder.yaml

```diff
@@ -88,8 +88,10 @@ paths:
       operationId: deleteFolder
       tags: [Folder]
       responses:
-        '204':
-          description: Success
+        '200':
```

Comment:
An async interface needs to have a result GETter.
You can see the result call [in clone](https://github.com/DataBiosphere/terra-workspace-manager/blob/5c408989e683410650440d85913e362bdb2269c0/openapi/src/parts/workspace.yaml#L156). This is the pattern and is specified in [MC Terra - Async REST API Specification](https://docs.google.com/document/d/1PTd4xvmV9xnEkWaIgFc6d3VUyJwwymQTFdhpKrdsUKw/edit#heading=h.ol9mx3vfhjjj).

Reply:
It would be good to return something with the 200 response, yes.